src/SerializeSourcemap.cc

```diff
@@ -182,6 +208,26 @@ sos::Object WrapPayloadSourcemap(const SourceMap<Payload>& payload)
     // Schema
     payloadObject.set(SerializeKey::Schema, WrapSourcemap(payload.schema));
 
+    // Content
+    sos::Array content;
+
+    /// Attributes
```

Comment:
I am not sure why you are using triple `/` in some cases, does it signalize something?